warehouse/router_identities.go

```diff
@@ -152,16 +149,16 @@ func (wh *HandleT) hasLocalIdentityData(warehouse model.Warehouse) (exists bool)
 `,
 		warehouseutils.IdentityMergeRulesTableName(warehouse),
 	)
-	err := wh.dbHandle.QueryRow(sqlStatement).Scan(&exists)
+	err := r.dbHandle.QueryRow(sqlStatement).Scan(&exists)
 	if err != nil {
 		// TODO: Handle this
 		panic(fmt.Errorf("Query: %s\nfailed with Error : %w", sqlStatement, err))
```

Comment:
For my information, how come we `panic` here but we are handling the other errors?

Reply:
A follow up task can also be created but first it'd be nice to understand the scope of this.